docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
       CONJUR_DATA_KEY: 'OyXV68Mip14xj33huGaQKewmmS+gKtDlp6ECZ2iATpU='
       DATABASE_URL: postgres://postgres@pg/postgres
       RAILS_ENV: development
-      CONJUR_AUTHENTICATORS: authn-ldap/test,authn-oidc/test,authn
+      CONJUR_AUTHENTICATORS: authn-ldap/test,authn-oidc/test,authn-jwt/test,authn
```

Comment:
This is only necessary if you are going to stand up an instance of the JWT authenticator and run integration tests against it in a client. It would be nice to see this if possible. We have been avoiding setting up e2e tests but if we could stand up a local instance in the docker environment it would be worthwhile to write some tests for the python client.

Reply:
I dont entirely understand what you mean by "going to stand up an instance", is this something that exists for other authenticators right now? 
should I add tests for this new authenticator in the OpenApi? 
As far as I know, as a team we want to do the minimum regarding the OpenApi at the moment